media/js/base/protocol/init-navigation.js

```diff
@@ -3,26 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 /**
- * Initialize Protocol language switcher.
+ * Initialize Protocol Navigation.
  */
 (function() {
     if (typeof Mzp === 'undefined' || typeof Mzp.Menu === 'undefined' || typeof Mzp.Navigation === 'undefined') {
         return;
     }
 
-    if (!Mzp.Menu.isSupported()) {
-        return;
-    }
-
-    var _mqWideNav = window.matchMedia('(min-width: 768px)');
+    var hasMediaQueries = typeof window.matchMedia !== 'undefined';
```

Comment:
I was more comfortable with `Mzp.Menu.isSupported`. That way if the menu's requirements change the check in init is still valid.

Reply:
I realized that `Menu` calls `isSupported()` during `init()` anyway, so I think this was somewhat redundant.